test/fixtures/json5/serialization/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
I placed it here because they are tests for json5 specification, byt i guess I should have placed them in `test/todo/json5/`